src/webhook/webhook.service.ts

```diff
@@ -1,73 +1,30 @@
 import { HttpService } from '@nestjs/axios';
 import { Injectable, NotFoundException } from '@nestjs/common';
-import { RecipeResponse } from 'src/recipe/responses/recipe.response';
+import { RecipeResponse } from '../recipe/responses/recipe.response';
 import { lastValueFrom } from 'rxjs';
 import { UpdateWebhookDTO } from './DTOs/update-webhook.dto';
 import { PrismaService } from '../prisma/prisma.service';
 import { WebhookResponse } from './responses/webhook.response';
-import { UserResponse } from 'src/user/responses/user.response';
+import { UserResponse } from '../user/responses/user.response';
+import { WebhookName } from './enums/webhookName.enum';
 
 @Injectable()
 export class WebhookService {
   constructor(
     private readonly httpService: HttpService,
     private readonly prismaService: PrismaService,
   ) {}
+  async sendWebhook(
```

Comment:
I would user generic type for this method
`async sendWebhook<T>(payload: T, name: WebhookName): Promise<void>`

https://www.typescriptlang.org/docs/handbook/2/generics.html